CITATION.cff

```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+title: gym_envs_urdf
```

Comment:
Could you change it to : Urdf-Environment
Then it would be consistent with the 4TU title and the License title.